ideconfigs/intellij-complex-nextraq-style.xml.xml

```diff
@@ -0,0 +1,608 @@
+<code_scheme name="NexTraqStyle" version="173">
+  <option name="OTHER_INDENT_OPTIONS">
+    <value>
+      <option name="CONTINUATION_INDENT_SIZE" value="4" />
+    </value>
+  </option>
+  <option name="ENABLE_SECOND_REFORMAT" value="true" />
+  <option name="KEEP_CONTROL_STATEMENT_IN_ONE_LINE" value="false" />
+  <option name="KEEP_BLANK_LINES_BEFORE_RBRACE" value="0" />
+  <option name="ALIGN_MULTILINE_PARAMETERS" value="false" />
+  <option name="ALIGN_MULTILINE_FOR" value="false" />
+  <option name="SPACE_BEFORE_ARRAY_INITIALIZER_LBRACE" value="true" />
+  <option name="CALL_PARAMETERS_WRAP" value="1" />
```

Comment:
![image](https://user-images.githubusercontent.com/61692292/235169785-c779db8e-6ea2-45d9-8efc-ec98b6af82e0.png)

Reply:
Here's the template text if you don't have it and want to add it:

public java.lang.String toString() {
    return new org.apache.commons.lang3.builder
        .ToStringBuilder(this, org.apache.commons.lang3.builder.ToStringStyle.SHORT_PREFIX_STYLE)
#foreach ($member in $members)
    .append("$member.name", $member.accessor)
#end
    .toString();
}